src/panels/config/integrations/integration-panels/ozw/ozw-network-dashboard.ts

```diff
@@ -129,6 +129,7 @@ class OZWNetworkDashboard extends LitElement {
                   <div class="card-actions">
                     ${this._generateServiceButton("add_node")}
                     ${this._generateServiceButton("remove_node")}
+                    ${this._generateServiceButton("cancel_command")}
```

Comment:
Should this always be enabled? Or only when we are adding/removing a node?

Reply:
It can also be used (if I understand correctly) to cancel other running commands on the network, so it would be best to be always available for now